.rubocop_rails.yml

```diff
@@ -91,3 +91,6 @@ Rails/SkipsModelValidations:
 Rails/Validation:
   Include:
     - decidim-*/app/models/**/*.rb
+
+Rails/CompactBlank:
```

Comment:
> This is disabled because compact_blank is a method available in later raiils versions ... and I got in a situation when I was trying to locally disable, but it errored with "Unecesary disable"... yet if i do not disable it, recommends to set:
> 
> ```
> translation.reject { |_k, v| v.blank? }
> ```
> 
> as
> 
> ```
> translation.compact_blank
> ```
> 
> Which will result in `NoMethodError`